Claudia/src/main/java/com/endava/claudia/task/four/CompanyOrder.java

```diff
@@ -0,0 +1,89 @@
+package com.endava.claudia.task.four;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Random;
+
+public class CompanyOrder implements Order {
+
+    public long id = 1;
+    public LocalDateTime dateTime;
+    public String company;
```

Comment:
Here you should have **Company company** so please create Company class for this!

Reply:
Done